src/app/inbound/mod.rs

```diff
@@ -1,13 +1,17 @@
+use super::spans::SpanConverter;
 use super::{classify, config::Config, dst::DstAddr, identity, DispatchDeadline};
 use crate::proxy::http::{
     client, insert, metrics as http_metrics, normalize_uri, profiles, router, settings,
     strip_header,
 };
 use crate::proxy::{accept, Server};
 use crate::transport::{self, connect, keepalive, tls, Connection};
-use crate::{core::listen::ServeConnection, svc, Addr};
+use crate::{core::listen::ServeConnection, svc, trace_context, Addr};
 use linkerd2_reconnect as reconnect;
+use opencensus_proto::trace::v1 as oc;
```

Comment:
Nit: it seems not great to me to make `app::inbound` and `app::outbound` be directly aware of OpenCensus — I think that ideally these modules would be written against interfaces that would abstract over the span collector. 

I'm not going to block the PR over this, but I'd like to hear if @olix0r has any thoughts.

Reply:
I agree. I'd prefer that we re-export any of the OC types we want from the linkerd-opencensus crate -- that should be a complete API.

Not a hard blocker but the work's not done until that's cleaned up, imo.